README.md

```diff
@@ -596,6 +598,39 @@ The DLP templates used in this tutorial are specifically tailored for inspecting
 2. [Create de-identification templates and run de-identification on sample data](https://cloud.google.com/dlp/docs/creating-templates-deid)
 
 
+## Run this pipeline outside Google Cloud shell
+
+Google Cloud Shell comes with a pre-configured development environment that includes many of the most popular tools and
+libraries, such as gcloud, gradle, java, git, vim, and more. This means that you don't have to spend time setting up 
+your own environment on your local machine. If you want to run this solution from your own machine, follow the 
+instructions given below.
+
+1. Create a new GCP project.
+2. Set up the tools and libraries required to run the DLP Dataflow pipelines including gradle, java, git, etc.
```

Comment:
Can we have a fixed list of the tools and libraries required including the versions required? I also see facing the below error -

 Detected Gradle 4.4.1, but jib requires Gradle 5.1 or higher.

Reply:
We can add a recommendation if you want but most of the things apart from gradle and java depends on the kind of environment the user is working on. For eg: jq is one such library that is required to run the setup script. This might not be pre-installed on some envs. The list can be very long so we need to be selective.

Reply:
It is a safe assumption to take that users will work with the latest versions available.

Reply:
@annadata @anurag-paliwal Thoughts?

Reply:
@chitara-01 I'd suggest to have a seperate requirements.txt or dependencies.txt file that can be referred to for outside cloudshell

Reply:
I think we should provide minimum setup instructions within this section:
- For Java - Link to openJDK version/setup instructions.
- For gradle - Provide instructions to set JAVA_HOME and point user to use bundled gradle wrapper (`./gradlew` or `./gradlew.bat`) which will install required gradle automatically.
- For other tools gcloud-cli/gcloud-sdk and jq: Just link to external installation instructions for them.

Reply:
Added the instructions. Please review. 
@anurag-paliwal @Goutam1511 @catherinerosesantos